packages/office-ui-fabric-react/src/components/Calendar/Calendar.tsx

```diff
@@ -120,9 +119,8 @@ export class Calendar extends BaseComponent<ICalendarProps, ICalendarState> impl
           styles.picker,
           styles.pickerIsOpened,
           styles.pickerIsFocused,
-          showSmallMonthPicker && ('is-monthPickerVisible ' + styles.pickerIsMonthPickerVisible),
-          isMonthPickerVisible && !isDayPickerVisible && !showMonthPickerAsOverlay && ('is-onlymonthPickerVisible ' + styles.pickerOnlyMonthPickerVisible),
-          showMonthPickerAsOverlay && styles.overlayedCalendars,
+          isMonthPickerVisible && ('ms-DatePicker-monthPickerVisible ' + styles.monthPickerVisible),
+          isMonthPickerVisible && isDayPickerVisible && ('ms-DatePicker-calendarsInline ' + styles.calendarsInline),
```

Comment:
Yes, they are not needed.  The month-only calendar was being styled with an additional set of styles (pickerOnlyMonthPickerVisible) that made it 'oversized', since we don't want the month-only calendar to be that large, I was able to remove those styles and simplify how all the styles were applied.